docs/source/testing/larva/options.rst

```diff
@@ -0,0 +1,265 @@
+Options
+========
+
+When creating Larva tests, you will find that tests always fail at certain points because there is a difference between the current result of the test and the stored result of a test, for example, the “timestamp”. After all, time changes every second. For solving this kind of issue, Larva has several options for ignoring /replacing parts of messages to compare them. This subsection is intended to name and describe all ignore, replace and other options within the Larva testing tool.
```

Comment:
Good that you try to write an introduction, but I would do this differently. Maybe you can start by repeating briefly what "read" and "write" steps do. This will help you later in the parts that now talk about the "stored result". I think "expected result" is a better phrase  Youu can introduce that phrase in your introduction.

I object to your statement here that Larva tests always fail at certain points. This is only true if you do not use ignores, but good Frank developers know about ignores. You can avoid this false statement when you start by repeating "read" and "write" steps.

Reply:
I think "read" and "write" steps should be introduced in a chapter which tells about the basic stuff of writing test scenario, here we are only talking about using ignores, and only step"read" is mentioned once.

Reply:
I like that your introduction is short. It explains why you would use ignores. I still object to your first sentence: "When creating Larva tests, you will find that some constantly changing values will make the tests fail at certain points because ...". The phrase "when creating Larva tests..." is general; both newbies and experienced Frank developers write them. But experienced Frank developers know about ignores. Experienced Frank developers will not have failing tests because of changing values, because they know how to use ignores.

I would avoid this semantic problem by writing something like: "As explained <reference to earlier section about Larva>, Larva tests verify that values read from the tested Frank configuration are equal to expected values. These expected values appear as constants in the Larva tests. However, some tested systems produce different results for each time that they are executed, even though the inputs are the same. Examples are systems that return a time stamp. In Larva, you can address this issue using *ignores*. With ignores, values returned from the tested system are transformed before they are compared to the expected value. As a Frank developer, you have to choose these transformations wisely. The tested value must be transformed to be constant, but you should keep meaningful aspects of your returned value to have a meaningful test. This section gives an overview of the transformations (ignores) that are available to you."

The above text is more wordy than your text. Can you write something that solves the semantic problem of your introduction, but that is less wordy than the above paragraph?

Reply:
Sorry, GitHub jeopardized my example text. I intended to write:

As explained &lt; Reference to earlier Larva section &gt;, Larva tests verify ...

Reply:
When you update your introduction as I suggest, you can also introduce the differences between ignores, replaces, removes and others. You can introduce your subsections.